allegro_test.go

```diff
@@ -32,7 +32,7 @@ func genRandomId() string {
 // put random data in
 // generate timestamp for time.now()
 func genRandomData() json.RawMessage {
-    rand.Seed(time.Now().UTC().UnixNano())
+    rand.Seed(time.Now().UTC().UnixNano() / 1000000)
```

Comment:
I read on stack overflow that 
time.Now().UnixNano() * (time.Nanosecond / time.Millisecond)
might be more future proof
https://stackoverflow.com/questions/24122821/go-golang-time-now-unixnano-convert-to-milliseconds